### PR TITLE
Don't include out params in "Add null check for all parameters"

### LINKED ISSUE #47030: Add null checks for all parameters adds check for out parameter
Animation

### PR DESCRIPTION
**Pull request by Youssef1313**
Fixes #47030

### CODE REVIEW

**Comment by Youssef1313** [comment]
@CyrusNajmabadi Is this ready to merge?

**Comment by jinujoseph** [comment]
@jasonmalinowski anything pending before this is ready for merge?

File: src/EditorFeatures/VisualBasicTest/InitializeParameter/AddParameterCheckTests.vb [hunk 1]
```diff
@@ -58,6 +58,25 @@ class C
...
+end class",
+"
+class C
+    public sub new(byref s as string)
+        If s Is Nothing Then
+            Throw New System.ArgumentNullException(NameOf(s))
+        End If
+    end sub
+end class")
+        End Function
```
Comment (CyrusNajmabadi) [inline src/EditorFeatures/VisualBasicTest/InitializeParameter/AddParameterCheckTests.vb:78]: can we have a VB test that at least documents the behavior with `<out>` params in vb?

Comment (jasonmalinowski) [reply src/EditorFeatures/VisualBasicTest/InitializeParameter/AddParameterCheckTests.vb:78]: Yeah, we probably should have this; that OK @Youssef1313?

**Comment by CyrusNajmabadi** [comment]
LGTM.  ping us when this passes and we can merge in.  Thanks!

**Comment by jasonmalinowski** [comment]
Thanks @Youssef1313 for the fix!

**Pull request merged**